lib/src/components/NxDropdownMenu/NxDropdownMenu.scss

```diff
@@ -68,6 +68,7 @@
 
   &:active {
     background-color: var(--nx-color-dropdown-menu-background-active);
+    color: var(--nx-color-text-stark);
```

Comment:
I'm not finding and example of the active state in the mockups, where is it?

Reply:
There are some discrepancies between Sketch and Figma(e.g. Figma is using rgba on the box-shadow value instead of hsla) which @JasonDSwearingen  is aware of and will take care of it, hence, will refer to sketch instead for this ticket.
However, there were issues with Sketch that it is kinda broken and the active state example is cut off, I checked with Jason and it seems only happened in the browser, not in the desktop app. He confirmed that the text color of active state would be `nx-grey-30` in light mode and `nx-grey-05` in dark mode.

Reply:
> He confirmed that the text color of active state would be nx-grey-30 in light mode and nx-grey-05 in dark mode.

I suspect you misspoke there as those numbers (the dark mode one in particular) don't make sense and don't match what you've implemented.  Could you clarify again what the active text colors are supposed to be?

Reply:
You are right, my brain prob was not clear at that moment.
The correct text color in Active state should be

- Light mode: `nx-grey-05`

- Dark mode: `nx-indigo-90`